blockindex/bloomfilterindexer.go

```diff
@@ -269,6 +260,7 @@ func (bfx *bloomfilterIndexer) commit(blockNumber uint64, blkBloomfilter bloom.B
 	return bfx.kvStore.WriteBatch(b)
 }
 
+// TODO: improve performance
```

Comment:
`// TODO (liuhaai): improve performance`

Reply:
IMO, this is a good independent task for beginners. I will file it as a good-frist-issue issue.